controllers/configurationpolicy_controller_test.go

```diff
@@ -754,3 +755,88 @@ func TestShouldEvaluatePolicy(t *testing.T) {
 		)
 	}
 }
+
+func TestShouldHandleSingleKeyFalse(t *testing.T) {
+	t.Parallel()
+
+	var unstruct unstructured.Unstructured
+	var unstructObj unstructured.Unstructured
+
+	var update, skip bool
+
+	tests := [][]map[string]interface{}{
```

Comment:
Instead of `map[string]interface{}` could you use a custom struct type defined in the body of the test? That way you get typing without having to type assert.